app/page/importToken/index.tsx

```diff
@@ -0,0 +1,129 @@
+import { useContext, useState } from "react"
+import { useHashLocation } from "wouter/use-hash-location"
+
+import { Button } from "~app/component/button"
+import { Divider } from "~app/component/divider"
+import { Input } from "~app/component/input"
+import { StepBackHeader } from "~app/component/stepBackHeader"
+import { ProviderContext } from "~app/context/provider"
+import { ToastContext } from "~app/context/toastContext"
+import { useAccount, useAction, useTokens } from "~app/hook/storage"
+import { Path } from "~app/path"
+import { getErc20Contract } from "~packages/network/util"
+import address from "~packages/util/address"
+import number from "~packages/util/number"
+
+export function ImportToken() {
+  const [, navigate] = useHashLocation()
+  const { provider } = useContext(ProviderContext)
+  const { setToast } = useContext(ToastContext)
+
+  const { importToken } = useAction()
+
+  const account = useAccount()
+  const tokens = useTokens()
+
+  const [tokenAddress, setTokenAddress] = useState("")
+  const [tokenAddressValid, setTokenAddressValid] = useState(false)
+
+  const [tokenSymbol, setTokenSymbol] = useState("")
+  const [tokenDecimals, setTokenDecimals] = useState(0n)
+  const [tokenBalance, setTokenBalance] = useState(0n)
+
+  const [tokenFetching, setTokenFetching] = useState(false)
+
+  const onTokenAddressChanged = async (value: string) => {
+    setTokenAddress(value)
+
+    // Clear token information
+    setTokenSymbol("")
+    setTokenDecimals(0n)
+    setTokenBalance(0n)
```

Comment:
how about we clear token information here if error occurs?

```
try {
 ....
} catch {
    // Clear token information
    setTokenSymbol("")
    setTokenDecimals(0n)
    setTokenBalance(0n)
 
}
```

Reply:
I wonder that if we clear them when error catched, when address is changed and waiting for fetching the new token info, the page would still present the previous token info.

Reply:
You're right. We should clear token information when the user is attempting to input new address.